tests/acceptance/features/webUISharingFilePermissionMultipleUsers/shareFileWithMultipleUsers.feature

```diff
@@ -10,29 +10,18 @@ Feature: Sharing files with multiple internal users with different permissions
       | username |
       | Alice    |
       | Brian    |
+      | Carol    |
 
   @issue-ocis-1743 @ocisSmokeTest
   Scenario Outline: share a file with multiple users with different roles and permissions
-    Given these users have been created with default attributes and without skeleton files in the server:
-      | username |
-      | user0    |
-      | Carol    |
-      | David    |
-    And user "Alice" has created file "lorem.txt" in the server
+    Given user "Alice" has created file "lorem.txt" in the server
     And user "Alice" has logged in using the webUI
     When the user opens the share dialog for file "lorem.txt" using the webUI
-
     And the user selects the following collaborators for the share as "<role>" with "<extra-permissions>" permissions:
       | collaborator | type |
-      | Regular User | user |
       | Brian Murphy | user |
       | Carol King   | user |
-      | David Lopez  | user |
-    And the user removes "David Lopez" as a collaborator from the share
-    And the user removes "Regular User" as a collaborator from the share
-    Then user "David Lopez" should not be visible in the collaborators selected options in the webUI
-    And user "Regular User" should not be visible in the collaborators selected options in the webUI
```

Comment:
The scenario is to share a file with multiple users so IMO these steps are irrelevant. I have added a new scenario to check the remove functionality of collaborators from the invite list